app/Http/Controllers/Users/UsersController.php

```diff
@@ -228,9 +227,7 @@ public function update(SaveUserRequest $request, $id = null)
         $permissions = $request->input('permissions', []);
         app('request')->request->set('permissions', $permissions);
 
-
-        $user = User::with('assets', 'assets.model', 'consumables', 'accessories', 'licenses', 'userloc')->withTrashed();
-        $user = Company::scopeCompanyables($user)->find($id);
+        $user = User::with('assets', 'assets.model', 'consumables', 'accessories', 'licenses', 'userloc')->find($id)->withTrashed();
```

Comment:
We need to swap `find` and `withTrashed` otherwise we end up with a query builder and not a user.

```suggestion
        $user = User::with('assets', 'assets.model', 'consumables', 'accessories', 'licenses', 'userloc')->withTrashed()->find($id);
```